sdk/textanalytics/Azure.AI.TextAnalytics/tests/samples/Sample6_RecognizePiiEntities.cs

```diff
@@ -22,10 +22,10 @@ public void RecognizePiiEntities()
             // Instantiate a client that will be used to call the service.
             var client = new TextAnalyticsClient(new Uri(endpoint), subscriptionKey);
 
-            string input = "A developer with SSN 859-98-0987 whose phone number is 206-867-5309 is building tools with our APIs.";
+            string input = "A developer with SSN 555-55-5555 whose phone number is 555-555-5555 is building tools with our APIs.";
```

Comment:
The service won't also change `Jenny` to `A developer` as part of redacting PII?  _Shame..._